src/codecovio.jl

```diff
@@ -47,13 +47,19 @@ module Codecov
     not already specified in args_array.
     """
     function set_defaults(args_array; kwargs...)
-        defined_names = [k for (k,v) in args_array]
+        defined_names = keys(pairs(args_array))
+        is_args_array = Pair{Symbol, Any}[]
+        if args_array isa Base.Iterators.IndexValue
```

Comment:
`IndexValue` isn't a thing on 0.6.

Reply:
yeah we should maybe have 2 versions of this function, a 0.7 version and a 0.6 version